rhba-dev-env/vagrant/provisioning/roles/setup_env/tasks/main.yml

```diff
@@ -0,0 +1,207 @@
+---
+- fail:
+    msg: "This role is only supported for Fedora hosts"
+  when: ansible_distribution != 'Fedora'
+
+- name: upgrade all packages
+  become: true
+  become_user: root
+  dnf:
+    name: "*"
+    state: latest
+- name: install the latest version of docker
+  become: true
+  become_user: root
+  dnf:
+    name: docker
+    state: latest
+
+- name: "docker storage configurations"
+  become: true
+  become_user: root
+  copy:
+    dest: /etc/sysconfig/docker-storage-setup
+    content: |
+          DEVS=/dev/sdb
+          VG=docker-vg
+
+- name: "docker storage setup"
+  become: true
+  become_user: root
+  command: docker-storage-setup 
+
+- name: "enable docker service"
+  become: true
+  become_user: root
+  service:
+   name: docker
+   enabled: yes
+
+- name: "start docker service"
+  become: true
+  become_user: root
+  service:
+   name: docker
+   state: started
+
+- name: ensure group "docker" exists
+  become: true
+  become_user: root
+  group:
+    name: docker
+    state: present
+
+- name: adding user vagrant to group docker
+  become: true
+  become_user: root
+  user:
+    name: vagrant
+    groups: docker
+    append: yes
+
+- name: add new docker configuration
+  become: true
+  become_user: root
+  template:
+    src: "registries.j2"
+    dest: /etc/containers/registries.conf
+
+- name: enable @cekit/cekit
+  become: true
+  become_user: root
+  command: dnf copr enable @cekit/cekit -y
```

Comment:
I think that is not necessary anymore.

Reply:
@spolti, Correct; cekit is a first-class dnf citizen now in fedora. I should note that we have to use 3.2.0, not 3.3.x, as that version introduced a breaking change that all modules need to have a version field defined in the yaml. And even if we do that in our modules, we are depending on cct_module and jboss-eap-modules modules that have not yet done that. So if there's a dnf way to specify "the latest 3.2.x version", that would be good for now.